cmd/nerdctl/build_test.go

```diff
@@ -45,6 +45,38 @@ CMD ["echo", "nerdctl-build-test-string"]
 	base.Cmd("run", "--rm", imageName).AssertOutContains("nerdctl-build-test-string")
 }
 
+func TestBuildLocal(t *testing.T) {
+	testutil.DockerIncompatible(t)
```

Comment:
why?

Reply:
Without it the test errored with not compatible.  I'm assuming that's because the regular `docker build` doesn't support "output" (it's in `buildx`).